### DISCUSSION TITLE
Using Wifi instead of bluetooth for watches that support wifi

### DISCUSSION THREAD

**Discussion by pleblancq** [house-of-abbey/GarminHomeAssistant#203] [Ideas]
Hi, what would be the effort to use wifi instead of bluetooth with watches that support it? 

It looks like the functions are pretty similar (even though the link is 4 years old) https://forums.garmin.com/developer/connect-iq/f/discussion/250924/returning-to-app-after-using-makewebrequest-using-sync---over-wi-fi-using-the-3-1-toybox-communications-syncdelegate

I would like to have to possibility to use wifi instead of my phone being the bridge.

Thanks

**Comment by lennarto** [comment]
That would really be awesome !! 
Also if the phone is not required at all, but I guess thats not possible.

**Reply by philipabbey** [reply]
Trying that out was a dog's dinner! See https://github.com/house-of-abbey/GarminHomeAssistant/issues/142. I think Wi-Fi is just set up for data synchronisation. I've given up on that idea.

**Comment by philipabbey** [comment]
See #255. We have a solution on its way.

**Reply by pleblancq** [reply]
Thanks!
